Software/Python/easygopigo3.py

```diff
@@ -476,6 +476,18 @@ def read_encoders(self):
 
         return encoders
 
+    def read_encoders_average(self):
+        """
+        | Reads the encoders' position in degrees. 360 degrees represent 1 full rotation (or 360 degrees) of a wheel.
+
+        :returns: the average of the two wheel encoder values
```

Comment:
Line
> :returns: the average of the two wheel encoder values

has to be 
> :returns: The average of the two wheel encoders values.

We did this way all over the documentation and this would "break" the "convention".